GNUmakefile

```diff
@@ -0,0 +1,76 @@
+# User-customizable variables:
+CPPFLAGS ?=
+CXX ?= c++
+CXXFLAGS ?=
+DEPFLAGS ?= -MD -MF $(@).d -MP -MT $(@)
+LDFLAGS ?=
+LDLIBS ?=
+LOADLIBES ?=
+OUTPUT_OPTION ?= -o $(@)
+TARGET_ARCH ?=
+build_dir = build
+
+.DELETE_ON_ERROR:
```

Comment:
Do we need these 3 lines? What do they do? Was is copy-pasted some kind of template?